src/WebJobs.Extensions.DurableTask/Microsoft.Azure.WebJobs.Extensions.DurableTask.Telemetry-net461.xml

```diff
@@ -0,0 +1,3721 @@
+<?xml version="1.0"?>
```

Comment:
The .Telemetry versions of these xmls are no longer required now that this is one single project and not a separate project.